src/librustdoc/clean/blanket_impl.rs

```diff
@@ -104,7 +104,7 @@ impl<'a, 'tcx> BlanketImplFinder<'a, 'tcx> {
                         // the post-inference `trait_ref`, as it's more accurate.
                         trait_: Some(clean_trait_ref_with_bindings(
                             cx,
-                            ty::Binder::dummy(trait_ref.skip_binder()),
+                            ty::Binder::dummy(trait_ref.subst_identity()),
```

Comment:
There are some `skip_binder` lower down in this function too. Should those be `subst_identity` as well?

Reply:
Yeah I believe so

Reply:
thanks, just updated the commit to add those :+1: